rust/src/lib.rs

```diff
@@ -462,29 +460,25 @@ pub unsafe extern "C" fn zcashlc_derive_transparent_address_from_public_key(
 /// call zcashlc_string_free with the returned pointer when done using it
 #[no_mangle]
 pub unsafe extern "C" fn zcashlc_derive_shielded_address_from_viewing_key(
-    extfvk: *const c_char,
+    ufvk: *const c_char,
```

Comment:
Maybe change the name to `zcashlc_derive_shielded_address_from_unified_viewing_key` so that callers are forced to change?

Reply:
I'm not aware of any view-only sapling wallets that use the SDK, but it will be problematic if there was. It's probably safe to do that change and respond to any issues filed about this API changes if so.

Reply:
Something it could be useful to keep around is sapling Spending Key helpers. If you have a guarda wallet, the current API can't support moving those funds out because it used to back up a sapling spending key instead of a seed phrase. 

Again, an edge case.

Reply:
Ideally we'd support that use case with a more comprehensive approach to spending key import. It could still be satisfied using USKs, using a Sapling-only USK.